### ISSUE TITLE
les données des tables attributeurs sont invisible  

### ISSUE THREAD

**Issue by juwizeye** [3liz/lizmap-web-client#2704]
Bonjour , 
J'ai installé Qgis server et lizmap sur une machine distant  et Qgis desktop en local sous Windows, cela marche bien je peux voir la carte .
![image](https://user-images.githubusercontent.com/96420889/149980877-b1abc00e-a999-4882-802b-2e1f7e457a27.png)
 mais j'ai un problème avec les données . Quand j'essaye de regarder les données d'une couche rien ne s'affiche. 
c'est pareil pour la localisation par couche.
![image](https://user-images.githubusercontent.com/96420889/149981153-b0aa87c7-769a-43b7-a5b5-708ae72691f9.png)

 J'ai testé en localhost ça marche bien .  
j'ai rien dans les logs de lizmap. Si vous avez un idée de comment corriger cet erreur je suis preneur 

J'utilise:
 - Lizmap version: 3.5.0
 - QGIS Server FULL version:3.16.16
 - QGIS Desktop FULL version:3.22.1
 - OS (Windows, Linux, MacOS, Android…): linux serveur distant , windows client local 
 - Browser (Firefox, Chrome…):
 - Lizmap admin information (users, logs…) stored into Sqlite (default)? oui 
 -WINSCP: pour synchroniser les fichiers . 
-répertoire local: c:\home\data\prod
-répertoire distant:/home/data/prod

**Comment by Gustry** [comment]
Do you have any error in the javascript console ? when clicking buttons ?

You should see network requests to fetch data.

**Comment by juwizeye** [comment]
@Gustry no i don't have any error

**Comment by juwizeye** [comment]
@Gustry  how I can i see network requests?

**Comment by juwizeye** [comment]
I have this error in nginx logs 
![image](https://user-images.githubusercontent.com/96420889/150826625-c88050bf-7cb0-4135-acc9-7101b394944c.png)

**Comment by juwizeye** [comment]
@Gustry , when I used chrome browser I found the following error : 
![image](https://user-images.githubusercontent.com/96420889/150942366-4e14d494-d27d-4dda-9b0f-54cacb08a9a1.png)

**Comment by Gustry** [comment]
Indeed, thanks to the URL you have published in #1714, we can see the GeoJSON when fetching data for the attribute table is invalid:
![Screenshot from 2022-01-26 10-10-43](https://user-images.githubusercontent.com/1609292/151135197-4804e70e-1564-421f-8bab-3eb8c0a5d4ae.png)

@nboisteault @mdouchin Any idea how this error is in the geojson data ?

**Comment by juwizeye** [comment]
@Gustry  is there any special configuration for Https? I think that the problem come from there because when I am on localhost it works fine.

**Comment by juwizeye** [comment]
I found the solution , I had used fcgiwrap to install qgisserver  and now I am using spawn-fcgi and it works fine. 
Thank you for your help @Gustry
